README.md

```diff
@@ -848,6 +848,17 @@ browser.frames # =>
 # ]
 ```
 
+### frame: `Frame | nil`
```

Comment:
@zw963 this place of documentation for [`lib/ferrum/page/frames.rb`](https://github.com/rubycdp/ferrum#frames) this class does not contain method `frame`